Chapter6.html

```diff
@@ -0,0 +1,189 @@
+<html>
+  <head>
+    <title>Tables</title>
+  </head>
+  <body>
+    <table>
+      <thead>
+        <tr>
+          <th></th>
+          <th scope="col">Home starter hosting</th>
+          <th scope="col">Premium business hosting</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th scope="row">Disk space</th>
+          <td>250mb</td>
+          <td>1gb</td>
+        </tr>
+        <tr>
+          <th scope="row">Bandwidth</th>
+          <td>5gb per month</td>
+          <td>50gb per month</td>
+        </tr>
+        <!-- more rows like the two above here -->
+      </tbody>
+      <tfoot>
+        <tr>
+          <td></td>
+          <td colspan="2">Sign up now and save 10%</td>
+        </tr>
+      </tfoot>
+    </table>
+
+    <table>
+      <tr>
+        <td>15</td>
+        <td>15</td>
+        <td>30</td>
+      </tr>
+      <tr>
+        <td>45</td>
+        <td>60</td>
+        <td>45</td>
+      </tr>
+      <tr>
+        <td>60</td>
+        <td>90</td>
+        <td>90</td>
+      </tr>
+    </table>
+
+    <table>
+      <tr>
+        <th></th>
+        <th scope="col">Saturday</th>
+        <th scope="col">Sunday</th>
+      </tr>
+      <tr>
+        <th scope="row">Tickets sold:</th>
+        <td>120</td>
+        <td>135</td>
+      </tr>
+      <tr>
+        <th scope="row">Total sales:</th>
+        <td>$600</td>
+        <td>$675</td>
+      </tr>
+    </table>
+
+    <table>
+      <tr>
+        <th></th>
+        <th>9am</th>
+        <th>10am</th>
+        <th>11am</th>
+        <th>12am</th>
+      </tr>
+      <tr>
+        <th>Monday</th>
+        <td colspan="2">Geography</td>
+        <td>Math</td>
+        <td>Art</td>
+      </tr>
+      <tr>
+        <th>Tuesday</th>
+        <td colspan="3">Gym</td>
+        <td>Home Ec</td>
+      </tr>
+    </table>
+
+    <table>
+      <tr>
+        <th></th>
+        <th>ABC</th>
+        <th>BBC</th>
+        <th>CNN</th>
+      </tr>
+      <tr>
+        <th>6pm-7pm</th>
+        <td rowspan="2">Movies</td>
+        <td>Comedy</td>
+        <td>News</td>
+      </tr>
+      <tr>
+        <th>7pm-8pm</th>
+        <td>Sport</td>
+        <td>Current Affairs</td>
+      </tr>
+    </table>
+
+    <table>
+      <thead>
+        <tr>
+          <th>Date</th>
+          <th>Income</th>
+          <th>Expenditure</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>1st January</th>
+          <td>250</td>
+          <td>36</td>
+        </tr>
+        <tr>
+          <th>2nd January</th>
+          <td>285</td>
+          <td>48</td>
+        </tr>
+     <!-- additional rows as above -->
+      <tr>
+        <th>31st January</th>
+        <td>129</td>
+        <td>64</td>
+      </tr>
+      </tbody>
+      <tfoot>
+        <tr>
+          <td></td>
+          <td>7824</td>
+          <td>1241</td>
+        </tr>
+      </tfoot>
+    </table>
+
+    <table width="400" Cellpadding="10 Cellspacing="5">
```

Comment:
This still doesn't look right. First, I don't think Cellpadding or Cellspacing are capitalized (no props ever are in HTML). Second, you're missing the closing quote on cellpadding.